sdk/identity/identity/src/credentials/chainedTokenCredential.ts

```diff
@@ -32,6 +33,10 @@ export class ChainedTokenCredential implements TokenCredential {
     let token = null;
     const errors = [];
 
+    const span = createSpan("ChainedTokenCredential-getToken", options);
+    options = modifySpanOptions(span, options);
```

Comment:
nit: personally, I think it's bad form to mutate function parameters. I prefer to treat them as immutable and create new local versions when necessary.